tabs/adjustments.js

```diff
@@ -57,14 +57,25 @@ TABS.adjustments.initialize = function (callback) {
         //
         // update selected function
         //
-        
+
         var functionList = $(newAdjustment).find('.functionSelection .function');
-        functionList.val(adjustmentRange.adjustmentFunction);
+        // update list of selected functions
+        var functionListOptions = $(functionList).find('option');
```

Comment:
You're doing this for every function. Would probably be more efficient to do this once outside of `addAdjustment`, and pass in the template.